lib/npg_pipeline/function/autoqc.pm

```diff
@@ -88,20 +89,22 @@ sub create {
   $self->info(sprintf 'Running autoqc check %s for run %i',
                       $self->qc_to_run(), $self->id_run());
 
+
   my @definitions = ();
-  foreach my $p ( $self->positions() ) {
-    my $is_multiplexed_lane = $self->is_multiplexed_lane($p);
-
-    my $h = {'id_run' => $self->id_run, 'position' => $p};
-    push @definitions, $self->_create_definition($h, $is_multiplexed_lane);
-
-    if ( $self->is_indexed && $is_multiplexed_lane) {
-      foreach my $tag_index (@{$self->get_tag_index_list($p)}) {
-        my %th = %{$h};
-        $th{'tag_index'} = $tag_index;
-        push @definitions, $self->_create_definition(\%th, $is_multiplexed_lane);
-      }
-    }
+  for my $dp (@{$self->products->{lanes}}) {
```

Comment:
Lane products might also be in data_products. Would it be better to create a list of unique data products and iterate over it? Otherwise, if we are not careful, some jobs might be created twice.